bool initialized
int32 trans_id
int32 version_id
float64 initial_start
float64 initial_goal
float64 start
float64 goal
float64 current_position
float64 current_velocity
float64 scale
float64 delta_goal_fit
lwr/Model lwr_model